packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -85,7 +85,8 @@ class InstrumentationRabbitMessage(RabbitMessageBase, NodeMessageBase):
     service_type: str
     service_key: str
     service_tag: str
-    result: Optional[RunningState] = None
+    result: RunningState | None = None
+    simcore_user_agent: str
```

Comment:
Q: is there a specific pattern to identify these agents (i.e. someting that can be more restrictive than ``str``)?

Reply:
currently there is only one of these. and that is only for puppeteer.
https://github.com/ITISFoundation/osparc-simcore/blob/580f666e01341a7d4d50f0c292c3f999d067c150/tests/e2e/tutorials/tutorialBase.js